src/test.tsx

```diff
@@ -30,6 +35,26 @@ describe(useErrorBoundary, () => {
     expect(componentDidCatch).toHaveBeenCalledTimes(1);
   });
 
+  it("wraps thrown values that are not Error instances with ReactUseErrorBoundaryWrappedError", () => {
```

Comment:
I'd appreciate a check here which looks at the WrappedError's extra attribute at the original data, just to be sure.

Other than that, seems great :)